spec/cli_spec.rb

```diff
@@ -16,8 +17,25 @@
     env = { 'TMPDIR' => tmpdir, 'FILENAME' => 'test.out' }
     cmd = %w[bundle exec rails-response-dumper]
     _stdout, stderr, status = Open3.capture3(env, *cmd, chdir: AFTER_HOOK_APP_DIR)
-    expect(stderr).to include('after hook error (RuntimeError)')
+    expect(stderr).to include <<~ERR
+      #{AFTER_HOOK_APP_DIR}/dumpers/after_hook_dumper.rb:8 after_hook.after_hook received after hook error
+      #{AFTER_HOOK_APP_DIR}/dumpers/after_hook_dumper.rb:9:in `block (2 levels) in <top (required)>'
+    ERR
     expect(status.exitstatus).to eq(1)
     expect(File.exist?("#{tmpdir}/#{env.fetch('FILENAME')}")).to eq(true)
   end
+
+  it 'outputs all errors after execution' do
+    env = { 'TMPDIR' => tmpdir, 'FILENAME' => 'test.out' }
```

Comment:
Are these environments variables necessary? They look specific to the `after_hook` app.

Reply:
nope :sweat_smile: